lib/libsys/shm_open.2

```diff
@@ -112,6 +112,22 @@ The size of the object can be adjusted via
 and queried via
 .Xr fstat 2 .
 .Pp
+On system supporting CHERI capabilities, capabilities may only be shared
+within a single address space unless the shared memory object is created
+with the
+.Dv O_SHARECAP
+flag in the initial call to
+.Fn shm_open .
+Additionally, capabilities are only allowed via mappings created using
+.Xr mmap 2
+where the
+.Dv PROT_CAP
+permission is specified.
```

Comment:
Maybe specify what happens if a capability is loaded or stored otherwise? "Allowed" is a bit unclear.